test/Proxy.js

```diff
@@ -0,0 +1,31 @@
+const { assert } = require('chai');
+const { ethers, upgrades } = require('hardhat');
+
+let proxy;
+let implementation;
+let GaslessV3Factory;
+let implementationAddress;
+
+describe('Proxy', function () {
+    before(async () => {
+        GaslessV3Factory = await ethers.getContractFactory('GaslessV3');
+        proxy = await upgrades.deployProxy(GaslessV3Factory, [
+            hre.network.config.config.wrappedTokenAddress,
+            hre.network.config.config.gasForSwap,
+            hre.network.config.config.gasForApproval,
+            hre.network.config.config.defaultGasPrice,
+        ]);
+        await proxy.deployed();
+        implementationAddress =
+            await upgrades.erc1967.getImplementationAddress(proxy.address);
+        implementation = GaslessV3Factory.attach(implementationAddress);
+    });
+
+    it('Works with Implemation_1', async function () {
```

Comment:
what is this test exactly checking?

Reply:
no need for this test, will remove it, my branch hai it's base from Jatin's branch, so this got imported